tests/instances.tcl

```diff
@@ -294,7 +292,7 @@ proc parse_options {} {
             incr j
             set ::host ${val}
         } elseif {$opt eq {--tls} || $opt eq {--tls-module}} {
-            package require tls 1.6
```

Comment:
I think we should not touch tls lib

Reply:
tls 1.6 requires TCL 8.x and it rejects TCL 9.x.

If we want to use TCL 9.0, we need to use tls 2.0, which is only a beta version now. It's not yet released.

If we want to allow different TCL versions, we also need different TLS lib versions, so I think we can remove the hard-coded version requirements.